src/kernel_config_manager.sh

```diff
@@ -43,6 +43,8 @@ function kernel_config_manager_main()
     exit 22 # EINVAL
   fi
 
+  [[ -n "${options_values['VERBOSE']}" ]] && flag='HIGHLIGHT_CMD'
```

Comment:
`flag` is setted by default to `SILENT` at line 22. Because of the "pattern" used by @aquilamacedo when [introducing verbose to the features](https://github.com/kworkflow/kworkflow/blob/b8129dc38e8c8f1ca1a3e29e787f9fa4c69d641a/src/self_update.sh#L14C1-L14C1), I think you could change line 22 to be only
```bash
local flag
```
and add the following line as the first command of the function
```bash
flag=${flag:-'SILENT'}
```